inc/sainexthopgroup.h

```diff
@@ -184,6 +184,41 @@ typedef enum _sai_next_hop_group_attr_t
      */
     SAI_NEXT_HOP_GROUP_ATTR_SELECTION_MAP,
 
+    /**
+     * @brief Adaptive routing and switching object for this group.
+     *
+     * @type sai_object_id_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_ARS
+     * @allownull true
+     * @default SAI_NULL_OBJECT_ID
+     */
+    SAI_NEXT_HOP_GROUP_ATTR_ARS_OBJECT_ID,
```

Comment:
Probably the question for SONIC folks. What is the trigger for binding the ARS object to NHG ? How is the ARS object chosen (multiple objects/profiles) can be created

Reply:
As discussed, this is part of SONiC HLD that is being worked on.